Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Base image to use for the final stage
-ARG base_image=public.ecr.aws/amazonlinux/amazonlinux:2
+ARG base_image=public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-nonroot:2021-12-01-1638322424
```

Comment:
Anything special about this particular tag? Or it's just the latest one?

Reply:
it's just latest one